client/fleeca.lua

```diff
@@ -10,37 +10,29 @@ local copsCalled = false
 local refreshed = false
 local currentLocker = 0
 
--- Handlers
+local function resetDoor(object, isBankOpen, closedHeading, openHeading)
```

Comment:
Consider passing in the bank 'object' instead, which contains isBankOpen, closedHeading, and openHeading.